climate_scanner/trends_innovation_classifier/notebooks/Prodigy_Recipes_Automation.ipynb

```diff
@@ -0,0 +1,471 @@
+       "      <td>Innovation</td>\n",
+       "      <td>Capsule wardrobe</td>\n",
+       "    </tr>\n",
+       "    <tr>\n",
+       "      <th>...</th>\n",
+       "      <td>...</td>\n",
+       "      <td>...</td>\n",
+       "    </tr>\n",
+       "    <tr>\n",
+       "      <th>994</th>\n",
+       "      <td>NaN</td>\n",
+       "      <td>NaN</td>\n",
+       "    </tr>\n",
+       "    <tr>\n",
+       "      <th>995</th>\n",
+       "      <td>NaN</td>\n",
+       "      <td>NaN</td>\n",
+       "    </tr>\n",
+       "    <tr>\n",
+       "      <th>996</th>\n",
+       "      <td>NaN</td>\n",
+       "      <td>NaN</td>\n",
+       "    </tr>\n",
+       "    <tr>\n",
+       "      <th>997</th>\n",
+       "      <td>NaN</td>\n",
+       "      <td>NaN</td>\n",
+       "    </tr>\n",
+       "    <tr>\n",
+       "      <th>998</th>\n",
+       "      <td>NaN</td>\n",
+       "      <td>NaN</td>\n",
+       "    </tr>\n",
+       "  </tbody>\n",
+       "</table>\n",
+       "<p>999 rows × 2 columns</p>\n",
+       "</div>"
+      ],
+      "text/plain": [
+       "    Innovation/Trend Specific Innovation/Trend\n",
+       "0         Innovation        3D printed clothes\n",
+       "1         Innovation               3D printing\n",
+       "2         Innovation      Autonomous transport\n",
+       "3         Innovation                    Biking\n",
+       "4         Innovation          Capsule wardrobe\n",
+       "..               ...                       ...\n",
+       "994              NaN                       NaN\n",
+       "995              NaN                       NaN\n",
+       "996              NaN                       NaN\n",
+       "997              NaN                       NaN\n",
+       "998              NaN                       NaN\n",
+       "\n",
+       "[999 rows x 2 columns]"
+      ]
+     },
+     "execution_count": 75,
+     "metadata": {},
+     "output_type": "execute_result"
+    }
+   ],
+   "source": [
+    "df_categories"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 19,
+   "id": "0e0d7649",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "def category_name_processing(category_name):\n",
+    "    category_name = category_name.lower()\n",
+    "    category_name = re.sub(\" \",\"_\",category_name)\n",
+    "    return category_name\n",
+    "\n",
+    "category_name_processing(\"3D printed clothes\")"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "c1d86bfe",
+   "metadata": {},
+   "source": [
+    "## General CONFIG"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 66,
+   "id": "ccb3486f",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "category_name = \"Artificial Intelligence\"\n",
+    "category_name_processed = category_name_processing(category_name)\n",
+    "log_path = \"/home/azureuser/christine/logs/\"\n",
+    "patterns_path = \"/home/azureuser/datadrive/code/christine/patterns/\"\n",
```

Comment:
Just one tiny change - to update the patterns path to patterns_path = "/home/azureuser/christine/patterns/"

Reply:
Ah okay and also may be the way we structured all config at the top.

I will make these two changes